lib/block-supports/layout.php

```diff
@@ -58,12 +58,12 @@ function gutenberg_get_layout_style( $selector, $layout ) {
 			$style .= '}';
 
 			$style .= "$selector > .alignwide { max-width: " . esc_html( $wide_max_width_value ) . ';}';
-
 			$style .= "$selector .alignfull { max-width: none; }";
 		}
 
 		$style .= "$selector .alignleft { float: left; margin-right: 2em; }";
 		$style .= "$selector .alignright { float: right; margin-left: 2em; }";
+		$style .= "$selector > * + * { margin-top: var( --wp--style--block-gap ); margin-bottom: 0; }";
```

Comment:
Not related to this PR but a possible improvement for the future: Rules like $selector .alignleft { float: left; margin-right: 2em; }, and $selector > * + * { margin-top: var( --wp--style--block-gap ); margin-bottom: 0; } etc don't rely on any user value. If we have 10 group blocks on a page all with the same layout we repeat these rules as inline styles 10 times. Could we just have a class with these styles not relying on user values one time and then just inject the class when needed? The way the styles would only be added one time.

Reply:
I thought about this a few time it would force us to have generic `.container-flow` classNames. Worth a try I think but we should be careful about these extra classes.